pkg/url/url.go

```diff
@@ -364,6 +436,82 @@ func List(client *occlient.Client, localConfig *config.LocalConfigInfo, componen
 	return urlList, nil
 }
 
+// ListDockerURL returns all Docker URLs for given component.
+func ListDockerURL(client *lclient.Client, componentName string, envSpecificInfo *envinfo.EnvSpecificInfo) (URLList, error) {
+	containers, err := dockerutils.GetComponentContainers(*client, componentName)
+	if err != nil {
+		return URLList{}, errors.Wrap(err, "unable to list component container")
+	}
+
+	localURLs := envSpecificInfo.GetURL()
+
+	var urls []URL
+
+	containerJSONMap := make(map[string]types.ContainerJSON)
+
+	for _, c := range containers {
+		var found = false
+		containerJSON, err := client.Client.ContainerInspect(client.Context, c.ID)
```

Comment:
Getting dejavu here, lot's of repeated code, perhaps we can throw these loops into a separate function so you don't have to copy-and-paste / maintain them twice?

Reply:
I've noticed that these two loops are similar; however, they are doing different things. 
The first loop is iterating through each container's config and find out if there is a match config in local env.yaml. If found a match, then the url is `Pushed`; if not, means the config does not exist in local env.yaml file,  then the url is `Locally Deleted`.

But we cannot simply use the first loop to determine if one url is "Not Pushed". Therefore we need the second loop. 

I will add more comments in the code to clarify the behavior & avoid confusion.